tests/e2e/default/enketo/submit-default-delivery-form.wdio-spec.js

```diff
@@ -10,7 +10,8 @@ const utils = require('../../../utils');
 const userData = require('../../../page-objects/default/users/user.data');
 const sentinelUtils = require('../../../utils/sentinel');
 
-describe('Submit Default Delivery Report', () => {
+// FLAKY: https://github.com/medic/cht-core/issues/8087
+describe.skip('Submit Default Delivery Report', () => {
```

Comment:
This one is failing a lot, even for other branches. It might be something that got more flaky with a different commit. I tried some quick fixes but didn't get it to be stable, so I'm taking the path of extreme prejudice to get the build reliably green, then we can fix the skipped ones over time.